library/src/main/java/com/squareup/timessquare/MonthView.java

```diff
@@ -25,17 +27,18 @@
   public static MonthView create(ViewGroup parent, LayoutInflater inflater,
       DateFormat weekdayNameFormat, Listener listener, Calendar today, int dividerColor,
       int dayBackgroundResId, int dayTextColorResId, int titleTextColor, boolean displayHeader,
-      int headerTextColor, Locale locale, DayViewAdapter adapter) {
+      int headerTextColor, boolean showDayNamesHeaderRowView, Locale locale,
+      DayViewAdapter adapter) {
     return create(parent, inflater, weekdayNameFormat, listener, today, dividerColor,
-        dayBackgroundResId, dayTextColorResId, titleTextColor, displayHeader, headerTextColor, null,
-        locale, adapter);
+        dayBackgroundResId, dayTextColorResId, titleTextColor, displayHeader, headerTextColor,
+        showDayNamesHeaderRowView, null, locale, adapter);
   }
 
   public static MonthView create(ViewGroup parent, LayoutInflater inflater,
       DateFormat weekdayNameFormat, Listener listener, Calendar today, int dividerColor,
       int dayBackgroundResId, int dayTextColorResId, int titleTextColor, boolean displayHeader,
-      int headerTextColor, List<CalendarCellDecorator> decorators, Locale locale,
-      DayViewAdapter adapter) {
+      int headerTextColor, boolean displayWeekView, List<CalendarCellDecorator> decorators,
```

Comment:
dude

Reply:
Thanks for the extra work! 🤕 sorry! I thought I had all of them.